### DISCUSSION TITLE
Gemini integration similar to the google app.

### DISCUSSION THREAD

**Discussion by Rendusaz** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#954] [Ideas]
On my android I got a notification if I wanted to use gemini or the google assistant I was thinking that maybe it might be possible to do that here ? Kinda wanna try but I don't wanna waste time if anybody knows that you can't let me know! I'd love to try. But I guess it would make the assistant kinda pointless anyways since gemini doesn't do home assistant stuff? Not too sure I have an android but iOS is my daily with my Mac and I love being able to control everything directly from my Mac... thank you very much developer it makes things a million times smoother.  youre the sh*t. I would have waited for Google to release an official app which they'd never do because everything is a bulls**t web app from them now, so yeah thanks again you're the man haha

**Comment by im-coder-lg** [comment]
So you want a Google Gemini desktop app?
I think that could be possible, but still, it might just as well be the same kind of implementation for GAUDC - electron + bumblebee(I believe it is bumblebee or porcupine for voice recog) + Assistant SDK(which might as well be dead)
It might be nice, but I don't see Gemini providing proper weather data like Assistant.
It might be the next big challenge, however. One that isn't for me as I'm not experienced in all this.
@Melvin-Abraham it's been years man, what do you think?

Also,
> youre the sh*t

Seems a bit rude, who are you scolding @Rendusaz?
